lib/job-iteration/csv_enumerator.rb

```diff
@@ -20,7 +20,7 @@ class CsvEnumerator
     #   csv = CSV.open('tmp/files', { converters: :integer, headers: true })
     #   JobIteration::CsvEnumerator.new(csv).rows(cursor: cursor)
     def initialize(csv)
-      unless csv.instance_of?(CSV)
+      unless csv.class.name == "CSV"
```

Comment:
Another option would be

```suggestion
      unless defined?(CSV) && csv.instance_of?(CSV)
```

or

```suggestion
      unless defined?(CSV) && CSV === csv
```

Reply:
Of course! I like that better.